docs/extensions.md

```diff
@@ -32,4 +32,15 @@ kwild --autogen -- --extension.streamr.node ws://localhost:7170 \
     --extension.streamr.target_db 0x1A58f48A0369656015D6BE305a3716F84F979A86:dimo_weather \
     --extension.streamr.target_procedure create_record \
     --extension.streamr.input_mappings param1:field1,param2:field2.field3
-```
\ No newline at end of file
+```
+
+## Supported Data Types
+
+The Streamr-Kwil extension natively supports the following JSON data types:
+
+- `string`
+- `number`
+- `boolean`
+- `array`
+
+To pass data to to a `uuid` or `uin256` column in Kwil, the data must be passed as a string. To pass data to a `blob` column, the data must be passed as a base64 encoded string and the schema should use the [`decode` function](https://docs.kwil.com/docs/kuneiform/functions#encoding-functions) to decode the data.
```

Comment:
Blob data doesn't have to be base64 encoded. It can be hex too. We should just say that it should be passed as an encoded string and decoded in the procedure.

Also, we should specify that array is only for any of the above types, and it cannot be an array of objects